src/components/Login/LoginPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {useEffect, useState} from 'react';
+import './LoginPage.css';
+
+function LoginPage(props) {
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [emailDirty, setEmailDirty] = useState('');
+    const [passwordDirty, setPasswordDirty] = useState('');
+    const [emailError, setEmailError] = useState('Email should not be empty');
+    const [passwordError, setPasswordError] = useState('Password should not be empty');
+    const [formValid, setFormValid] = useState(false);
+
+    useEffect(() => {
+        if(emailError || passwordError) {
+            setFormValid(false);
+        } else {
+            setFormValid(true);
+        }
+    }, [emailError, passwordError])
+
+    const emailHandler = (e) => {
+        setEmail(e.target.value);
+        const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+        if(!re.test(String(e.target.value).toLowerCase())){
+            setEmailError('Incorrect email')
+        } else {
+            setEmailError('')
+        }
+    }
+
+    const passwordHandler = (e) => {
+        setPassword(e.target.value);
```

Comment:
if(!e.target.value) {
                setPasswordError('Password should not be empty');
            }